Gopkg.toml

```diff
@@ -29,7 +29,7 @@ required = [
 
 [[constraint]]
   name = "github.com/gogo/protobuf"
-  version = "=v1.2.1"
+  version = "=v1.3.0"
```

Comment:
What's this dependency bump for? Just making sure we don't run into any incompatibility issues.

Reply:
Just matching protobuf version in gravity https://github.com/gravitational/gravity/blob/95d4167587056e66cfecc55e391f4bff29d55481/Gopkg.toml#L74